gelato-relay/bin/status.rs

```diff
@@ -0,0 +1,14 @@
+use gelato_relay::*;
+
+#[tokio::main]
+async fn main() -> Result<(), reqwest::Error> {
+    let gelato = GelatoClient::default();
+
+    let task_status = gelato
+        .get_task_status("0x6265e895da9daf43ad028392669809127e4524760528488953e775f2be2b2b3e")
+        .await
+        .unwrap();
+    println!("Task status: {:?}", task_status);
```

Comment:
Do we want a `println` here?

Reply:
Yes this is just a script to print task status. Ideally should be CLI command where we pass `task id` but not too important.`bin/status.rs` is really just sanity checking tool I used when testing locally but want to keep because it helps for debugging